projects/chaos/inc/chaos_events.h

```diff
@@ -34,19 +32,35 @@ typedef enum {
   CHAOS_EVENT_GPIO_DRIVE,
   CHAOS_EVENT_CHARGER_CLOSE,
   CHAOS_EVENT_CHARGER_OPEN,
-  NUM_CHAOS_EVENTS_FSM  // 21
+  NUM_CHAOS_EVENTS_FSM  // 19
 } ChaosEventsFSM;
 
 // Used for the sequence generator that schedules and order events for the other FSMs.
+//
+// These are lower priority than other events because the other events must complete first otherwise
```

Comment:
No longer accurate? Now that the event priority isn't encoded in the ID anymore we should probably mention if specific events are meant to be transmitted with non-default priorities.